R/radial_bridge.R

```diff
@@ -67,7 +67,24 @@ read_h5 <- function(file, name = NULL, ...) {
 #' }
 #' }
 
-tidy.brain <- function(x, threshold = 0.9, ...) {
+tidy.brain <- function(x, type="wildtype", threshold.n=0.9, ...) {
+  if (type=="youtoo"){
+    if(threshold.n != 0.9){
+      threshold = threshold.n
+      print(paste0("Set threshold: ", threshold))
```

Comment:
Please do not use `print()`. Use `message()`. This will allow the user to suppress the messages if they wish.